numpy/_core/_add_newdocs.py

```diff
@@ -1698,7 +1698,7 @@
 
 add_newdoc('numpy._core.multiarray', 'arange',
     """
-    arange(start, / [, stop[, step,], dtype=None, *, device=None,, like=None)
+    arange(start, [, stop[, step,], dtype=None, *, device=None,, like=None)
```

Comment:
I am not sure which of these makes the most sense. (1) is the signature in 1.26, (2) is from #25336, or does (3) make sense?

1. `arange([start,] stop[, step,], dtype=None, *, device=None, like=None)`
2. `arange(start, / [, stop[, step,], dtype=None, *, device=None,, like=None)`
3. `arange(start, [, stop[, step,], dtype=None, *, device=None,, like=None)`

Reply:
I don't think 3 does since it would look like `start=` on its own makes sense.

I'm fine with either 1 and 2.  We could write `step=1` (should be right).  Then the 2. gives a valid signature, but lies, while the first can't be represented by Python signatures nicely but captures the keyword usage...

Reply:
I will stick with (1) then. I don't know what you mean by "python signatures, but the arg checking can be done in pure python:
```
missing = object()
def arange(start, stop=missing, step=1, dtype=None, *, device=None, like=None):
    if stop is missing:
        stop = start
        start = 0
```